charts/vsphere-cpi/v1.0.0/templates/vsphere-cpi-ds.yaml

```diff
@@ -22,8 +22,8 @@ spec:
         k8s-app: vsphere-cloud-controller-manager
     spec:
       nodeSelector:
-      {{- if .Values.nodeSelector}}
-      {{- with .Values.nodeSelector }}
+      {{- if .Values.cloudControllerManager.nodeSelector }}
+      {{- with .Values.cloudControllerManager.nodeSelector }}
         {{- toYaml . | nindent 8 }}
       {{- end }}
```

Comment:
Does this render correctly when you try to add nodeSelectors on the values.yaml and run `helm template`? From my experience, you usually have to unindent these, but I could be wrong

Reply:
This was a change made by Rajashree so I assume it works. I'll test it to make sure.